core/api/tests/test_projects_v2.py

```diff
@@ -117,7 +118,7 @@ def setup_project_list(
             "status": project_status,
             "submission_status": project_submission_status,
             "sector": sector,
-            "subsector": subsector,
+            "subsectors": [subsector],
             "substance_type": "HCFC",
```

Comment:
Should modify for setup_project_create

Reply:
this was actually `"subsector_ids": [subsector.id]`